packages/website/pages/docs.js

```diff
@@ -29,48 +36,52 @@ export default class Docs extends Component {
         <NavigationWrapper navContent={DocsNavContent}>
           <Page>
             <Title>Documents Overview</Title>
-            <Section>
-              <TableTree>
-                <Headers>
-                  <Header width={300}>Name</Header>
-                  <Header width={400}>Path</Header>
-                </Headers>
-                <Rows
-                  items={pageInfo.docs}
-                  render={({ id, pagePath, children }) => (
-                    <Row
-                      itemId={id}
-                      items={children}
-                      hasChildren={children && children.length > 0}
-                      isExpanded={Boolean(expansionMap[id])}
-                      onExpand={() =>
-                        this.setState({
-                          expansionMap: {
-                            ...expansionMap,
-                            [id]: true,
-                          },
-                        })
-                      }
-                      onCollapse={() =>
-                        this.setState({
-                          expansionMap: {
-                            ...expansionMap,
-                            [id]: false,
-                          },
-                        })
-                      }
-                    >
-                      <Cell singleLine>{titleCase(id)}</Cell>
-                      <Cell>
-                        <Link href={pagePath}>
-                          <a>{pagePath}</a>
-                        </Link>
-                      </Cell>
-                    </Row>
-                  )}
-                />
-              </TableTree>
-            </Section>
+            {pageInfo.docs ? (
+              <Section>
```

Comment:
How much of a pain would it be to extract this as its own component, so that we get a really concise ternary that's a bit easier to read?